tests/test_models/test_engine/test_file_storage.py

```diff
@@ -113,3 +113,18 @@ def test_save(self):
         with open("file.json", "r") as f:
             js = f.read()
         self.assertEqual(json.loads(string), json.loads(js))
+
+    @unittest.skipIf(models.storage_t == 'db', "not testing file storage")
+    def test_get(self):
+        """Test that get properly returns a requested object"""
+        storage = FileStorage()
+        user = User(name="User1")
+        user.save()
+        self.assertEqual(user, storage.get("User", user.id))
+
+    @unittest.skipIf(models.storage_t == 'db', "not testing file storage")
+    def test_count(self):
+        """Test that count properly counts all objects."""
+        storage = FileStorage()
```

Comment:
# testing coverage of all the codes and the new methods added. we ensure that the changes and additions made to the codes don't alter anything... and this is the whole essence of the unit testing